content/docs/user-guide/scripting/script-canvas/programmer-guide/custom-nodes/custom-node-definition-reference.md

```diff
@@ -0,0 +1,382 @@
+| __Uuid__ | Optional | Allows you to provide a specific Universal Identifier, this is generally generated but in some situations it may be desirable to provide a specific UUID.  | `Uuid="{EE36A690-7C33-445F-B9E8-BD045D6ACC1D}"` |
+| __Icon__ | Unused | Currently unused, would provide a path for displaying a custom icon associated with a node | `Icon="Icons/ScriptCanvas/Checkpoint.png"` |
+| __Version__ | Optional | As with any serialized class in O3DE, nodes may provide a version, this creates a mechanism for addressing code changes that may invalidate data | `Version="2"` |
+| __VersionConverter__ | Optional | If the topography of a node changes, a version converter may be used to provide a migration mechanism | `VersionConverter="ScriptCanvas::ForEachVersionConverter"` |
+| __EventHandler__ | Optional | Used by the serialization system, it may sometimes be useful to capture serialization events| `EventHandler="SerializeContextOnWriteEndHandler&lt;EBusEventHandler&gt;"` |
+| __Deprecated__ | Optional | Marks this node as deprecated, deprecated nodes will not be displayed in the Node Palette, but may still appear in existing Script Canvas graphs | `Deprecated="This node has been deprecated"`|
+| __DeprecationUUID__ | Optional | When specified, this UUID is used to replace this deprecated node with the node of the provided UUID, this allows graphs to be updated automatically | `DeprecationUUID="{D3629902-02E9-AE59-0424-F366D342B433}"` |
+| __Base__ | Optional | In some cases it may be desireable to specify the base class, for example when a node derives from a different node | `Base="ScriptCanvas::Nodes::Internal::BaseTimerNode"` |
+| __GraphEntryPoint__ | Optional | When True, this node may be used by Script Canvas as the place where execution will begin | `GraphEntryPoint="True"` |
+| __EditAttributes__ | Optional | Allows the ability to provide __EditContext__ attributes in the node's serialization, use the @ character to separate the key/value and separate multiple attributes with a semi colon; | `EditAttributes="AZ::Script::Attributes::ExcludeFrom@AZ::Script::Attributes::ExcludeFlags::All;` `ScriptCanvas::Attributes::Node::TitlePaletteOverride@StringNodeTitlePalette"` |
+| __DynamicSlotOrdering__ | Optional | Should be used when the order of slots may change at edit time | `DynamicSlotOrdering="True"` |
+----
+         
+
+
+### Examples
+
+```xml
+<ScriptCanvas Include="Include/ScriptCanvas/Libraries/Core/Start.h" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <Class Name="Start"
+        QualifiedName="ScriptCanvas::Nodes::Core::Start"
+        PreferredClassName="On Graph Start"
+        Uuid="{F200B22A-5903-483A-BF63-5241BC03632B}"
+        Category="Timing"
+        Version="2"
+        GraphEntryPoint="True"
+        Description="Starts executing the graph when the entity that owns the graph is fully activated.">
+
+</Class>
+```
+
+
+## Execution
+
+### <a name="input"></a>Input
```

Comment:
Was this anchor not used? I think I meant to map it from some other location for easy reference.

Reply:
Hugo doesn't support HTML anchor elements. By default, it will create an anchor from the text in each header automatically. If you want to override that with a custom-defined anchor, the format to use after the header text is `{#my-anchor}`.

Reply:
I tested them and was able to use them

Reply:
What did you test, may I ask, and what do you mean when you say you were able to use them?

For example, the link for this specific header is rendered as https://deploy-preview-1984--o3deorg.netlify.app/docs/user-guide/scripting/script-canvas/programmer-guide/custom-nodes/custom-node-definition-reference/#a-nameinputainput.

The anchor at the end there is `#a-nameinputainput`. While technically that is a working link if you click on it from my comments here, it is needlessly complex. If you had just left the header as `## Input`, the anchor would simply be `#input`, which is preferable.

Furthermore, the places where you are trying to use this link are not working properly with your HTML anchor. For example, look at this sentence:

> Parameter is used by the [Input](https://deploy-preview-1984--o3deorg.netlify.app/docs/user-guide/scripting/script-canvas/programmer-guide/custom-nodes/custom-node-definition-reference/#input) and [Output](https://deploy-preview-1984--o3deorg.netlify.app/docs/user-guide/scripting/script-canvas/programmer-guide/custom-nodes/custom-node-definition-reference/#output) elements, they represent data slots on the node.

Click on the **Input** link and you'll see that, while it does scroll to a different place on the page, it's not going to the correct place. It scrolls all the way up to the top. This is because it can't find any `#input` anchor.